packages/tinacms/src/tina-cms.tsx

```diff
@@ -200,6 +241,20 @@ export const TinaCMSProvider2 = ({
   if (typeof props.query === 'string') {
     props.query
   }
+
+  // branch & clientId are still supported, so don't throw if they're provided
+  if (!props.apiURL || (props.branch && props.clientId)) {
```

Comment:
As is, won't this make things not work for anyone using props.branch && props.clientId instead of apiURL? Should it be:
```suggestion
  if (!props.apiURL || ( !props.clientId && !isLocalClient )) {
```